### PR TITLE
Update of DruckerPrager implementation.

### PR DESCRIPTION
**Pull request by endJunction**

### CODE REVIEW

**Comment by endJunction** [comment]
@nagelt From `beta=1/6` and `kappa~=0.0866=cos(30)/10` I find through {`kappa=c*cos(phi)`; `beta=1/3 sin(phi)`} that `phi=30` and `c=1/10`, being Internal friction parameter and cohesion respectively. 
How to find the dilatancy parameter? If it is same as `phi`, then something is different from the Ehlers model in the DruckerPrager benchmark https://github.com/ufz/ogs/blob/master/Tests/Data/Mechanics/Ehlers/MFront/cube_1e0_dp.prj.
(Tried both, phi in degree and rad). Also tried with dilatancy set zero; it's not matching the reference solutions.

**Comment by nagelt** [comment]
@endJunction I only tested it with the DruckerPrager test for MFront. There, kappa (old file) = cohesion (new file), beta (old file) = DilatancyParameter=FrictionParameter.
For Ehlers you also don't have to re-calculate anything. Kappa = cohesion and beta = FrictionParameter, beta_p = DilatancyParameter. The latter I have not checked myself ...

File: Tests/Data/Mechanics/Ehlers/MFront/cube_1e0_dp.prj [hunk 1]
```diff
@@ -13,8 +13,9 @@
                 <material_properties>
                     <material_property name="YoungModulus" parameter="E"/>
                     <material_property name="PoissonRatio" parameter="nu"/>
-                    <material_property name="beta" parameter="beta"/>
-                    <material_property name="kappa" parameter="kappa"/>
+                    <material_property name="Cohesion" parameter="kappa"/>
+                    <material_property name="FrictionParameter" parameter="beta"/>
+                    <material_property name="DilatancyParameter" parameter="beta"/>
```
Comment (TomFischer) [inline Tests/Data/Mechanics/Ehlers/MFront/cube_1e0_dp.prj]: Is it correct that FrictionParameter and DilatancyParameter both use the parameter beta?

Comment (endJunction) [reply Tests/Data/Mechanics/Ehlers/MFront/cube_1e0_dp.prj]: yes, that's correct.

**Comment by endJunction** [comment]
Unrelated test failed...

**Pull request merged**